content/docs/command-reference/fetch.md

```diff
@@ -56,14 +46,25 @@ Here are some scenarios in which `dvc fetch` is useful, instead of pulling:
 - After checking out a fresh copy of a <abbr>DVC repository</abbr>, to get
   DVC-tracked data from multiple project branches or tags into your machine.
 - To use comparison commands across different Git commits, for example
-  `dvc metrics show` with its `--all-branches` option.
+  `dvc metrics show` with its `--all-branches` option, or `dvc plots diff`.
 - If you want to avoid [linking](/doc/user-guide/large-dataset-optimization)
   files from the cache, or keep the <abbr>workspace</abbr> clean for any other
   reason.
 
+Without arguments, it downloads all files and directories referenced in the
+current workspace (found in `dvc.yaml` and `.dvc` files) that are missing from
+the workspace. Any `targets` given to this command limit what to fetch. It
+accepts paths to tracked files or directories (including paths inside tracked
+directories), `.dvc` files, and stage names (found in `dvc.yaml`).
+
+The `--all-branches`, `--all-tags`, and `--all-commits` options enable fetching
+files/dirs referenced in multiple Git commits.
+
 The default remote is used (see
-[`dvc config core.remote`](/doc/command-reference/config#core)) unless the
-`--remote` option is used.
+[`dvc config core.remote`](/doc/command-reference/config#core)) unless a
+specific one is given with `--remote`.
+
+> See `dvc gc` for ways to remove cached files.
```

Comment:
it's a strange reference, why `gc`, why not checkout, not pull, other commands ... or remove?

Reply:
checkout and pull are referenced near the top of the description. This note is the very last thing in the description.

Reply:
I understand, but it feels very artificial to me ... probably because we don't do this in other place (like `dvc pull`?)

Reply:
Yeah. It's not very necessary. Removed.